package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Guest Management System",
   "main": "index.js",
   "scripts": {
+    "make-guest": "npx sequelize-cli model:generate --name guest --attributes name:string,email:string,phone:string,gender:string",
```

Comment:
Why did you add this script: make-guest?

Use this instead for migrate and down. 
db:migrate": "sequelize db:migrate",
"db:undo:migrate": "sequelize db:migrate:undo",
"db:seed": "sequelize db:seed:all",

its close to the original syntax, that execute.

You don't need the npx, you already using yarn.

Reply:
Okay. I have resolved that.